src/modal/modal.spec.ts

```diff
@@ -248,6 +249,38 @@ describe('ngb-modal', () => {
         expect(fixture.nativeElement).not.toHaveModal();
       });
 
+      it('should dismiss with dismissAll', fakeAsync(() => {
```

Comment:
This tests go through low-level APIs `NgbModalStack` and asserts on "private" (AFAIK) API (`modalRefs`).

I would prefer this test to be written as all the existing tests and go through  `NgbModal` - just open one modal, call `dismissAll(...)` on `NgbModal` and assert that we don't have any modals left in the DOM. We don't need much more for now, until we support stacked modals.